src/work-allocation-2/containers/task-assignment/task-assignment-container.component.spec.ts

```diff
@@ -98,7 +98,9 @@ describe('TaskAssignmentContainerComponent', () => {
 
     wrapper.tasks = null;
     window.history.pushState({ returnUrl: 'mywork/list', showAssigneeColumn: false }, '', 'mywork/list');
-    fixture.detectChanges();
+    // Deliberately defer fixture.detectChanges() call to each test, to allow overriding the ActivatedRoute snapshot
+    // data with a different verb ("Assign")
+    // fixture.detectChanges();
```

Comment:
could these be removed?

Reply:
@rayliangatos - happy to remove Line 103 but I think I'll leave the comment in place to explain why `fixture.detectChanges()` isn't being called here (which it would usually).